backend/app/src/main/java/com/moh/phlat/backend/esb/json/MaintainHdsResponse.java

```diff
@@ -14,101 +12,110 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import com.moh.phlat.backend.model.Message;
 import com.moh.phlat.backend.model.ProcessData;
+import com.moh.phlat.backend.service.MessageSourceSystem;
 
+import io.github.resilience4j.circuitbreaker.CallNotPermittedException;
 import lombok.Getter;
 
 public class MaintainHdsResponse implements PlrResponse {
 	private static final Logger logger = LoggerFactory.getLogger(MaintainHdsResponse.class);
 	
-	private ProcessData data;
+	private static final String HDS_LOADED_RESPONSE_CODE = "PRS.PRV.OID.CRE.0.0.0003";
+
+	private static ObjectMapper objectMapper;
+	static {
+		JSON_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
+		
+		objectMapper = new ObjectMapper();
+		objectMapper.setDateFormat(JSON_DATE_FORMAT);
+		objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+		objectMapper.disable(SerializationFeature.WRITE_DATES_WITH_ZONE_ID);
+		objectMapper.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
+		objectMapper.setSerializationInclusion(Include.NON_NULL);
+	}
 	
-	@Getter
-	private String hdsId;
+	private ProcessData data;
 	
 	@Getter
-	private boolean isLoaded = false;
-	@Getter
-	private boolean isDuplicate = false;
-	private boolean hasError = false;
-	
-	private List<PlrError> plrErrors = new ArrayList<>();
+	private boolean loaded = false;
 	
 	public MaintainHdsResponse(ProcessData data) {
 		this.data = data;
 	}
 	
 	@Override
 	public void plrJsonToProcessData(String json) {
```

Comment:
```suggestion
	public void HDSResponseToProcessData(String hdsJsonResponse) {
```

PLR Json is too generic name